README.md

```diff
@@ -16,7 +16,7 @@ https://www.opencontainers.org/) solution.
 `cc-proxy` is a daemon offering access to the [`agent`](
 https://github.com/clearcontainers/agent) to both the runtime and shim
 processes. Only a single instance of `cc-proxy` per host is necessary as it can
-be used for several different VMs.
+be used for several different VMs. However, since [Clear Containers 3.0.10](https://github.com/clearcontainers/runtime/releases/tag/3.0.10) one proxy instance per virtual machine is launched for improved isolation.
```

Comment:
Thought I'd throw this in for fun for @jodh-intel :-)
http://www.quickanddirtytips.com/education/grammar/starting-a-sentence-with-however-right-or-wrong

Reply:
However, I have however update the branch. However.